docs/imviz/plugins.rst

```diff
@@ -318,6 +318,8 @@ right or the left, as well as a slider and input to set the angle and a switch t
 axes should be flipped horizontally after applying the rotation (a vertical flip can be achieved
 via a 180 deg rotation and a horizontal flip).
 
+Due to known issues, Canvas Rotation is only available on Chromium-based browsers.
```

Comment:
Maybe make this a `..warning:` section?

Reply:
And maybe "Due to browser limitations" instead of "known issues" (so it sounds like its less our fault 😉 )

Reply:
Done in https://github.com/spacetelescope/jdaviz/pull/2192/commits/ea90a70fc11fff01312ff22c0f6ee269ef968078